src/main/java/view/OutputView.java

```diff
@@ -0,0 +1,79 @@
+package view;
+
+import model.player.Player;
+import model.ladder.LadderRow;
+import model.player.PlayerResults;
+
+import java.util.List;
+import java.util.Map;
+
+public class OutputView {
+    private static final int COLUMN_LENGTH = 5;
+
+    private void outputPlayer(List<Player> playerInventory) {
+        for (Player player : playerInventory) {
+            String playerName = player.getName();
+            System.out.printf("%" + COLUMN_LENGTH + "s", playerName);
+            System.out.printf(" ");
+        }
+    }
+
+    private void outputLadderRow(List<Boolean> points) {
+        System.out.println();
+        System.out.print("    |");
+        for (int i = 0; i < points.size(); i++) {
+            outputLadderColumn(points.get(i));
+        }
+    }
+
+    private void outputLadderColumn(Boolean bool) {
+        if (bool == true) System.out.print("-----|");
+        if (bool == false) System.out.print("     |");
+    }
+
+    public void outputLadder(List<Player> playerInventory, List<LadderRow> ladderRows, List<String> trialResults) {
+        System.out.println("\n" + "사다리 결과" + "\n");
+        outputPlayer(playerInventory);
+        for (int i = 0; i < ladderRows.size(); i++) {
+            List<Boolean> points = ladderRows.get(i).getPoints();
+            outputLadderRow(points);
+        }
+        outputResults(trialResults);
+        System.out.println();
+    }
+
+    private void outputResults(List<String> trialResults) {
+        System.out.println();
+        for (String trialResult : trialResults) {
+            System.out.printf("%" + COLUMN_LENGTH + "s", trialResult);
+            System.out.printf(" ");
+        }
+    }
+
+    private void outputAllPlayersResult(PlayerResults playerResults) {
+        Map<String, String> PlayerResultsInventory = playerResults.getPlayerResultsInventory();
+        for (Map.Entry<String, String> stringStringEntry : PlayerResultsInventory.entrySet()) {
+            String playerName = stringStringEntry.getKey();
+            String playerResult = stringStringEntry.getValue();
+            System.out.println(playerName + " : " + playerResult);
+        }
+    }
+
+    private void outputCertainPlayerResult(PlayerResults playerResults, String playerName) {
+        String playerResult = playerResults.getPlayerResult(playerName);
+        if (playerResult == null) {
+            System.out.println("존재하지 않는 사용자입니다.");
+        } else if (playerResult != null) {
+            System.out.println(playerName + " : " + playerResult);
+        }
```

Comment:
```suggestion
        if (playerResult == null) {
            System.out.println("존재하지 않는 사용자입니다.");
            return
        }
        System.out.println(playerName + " : " + playerResult);
```
개인 취향은 이렇게 쓰는 편인데 어떻게 생각하시나요?

Reply:
else를 쓰면 안 된다는 조건이 있어서 else if를 사용하긴 했는 데 리뷰어님께서 추천해주신 방법이 더 깔끔해보이네요!!
리뷰어님의 추천대로 코드를 수정하였습니다!!
앞으로도 elseif가 하나일때는 return을 써보는 방식을 사용해야겠어요!! 감사합니다 :)